Source/WTF/Scripts/GeneratePreferences.rb

```diff
@@ -266,6 +291,16 @@ def initializeParsedPreferences(parsedPreferences)
           next
         end
 
+        if %w{ developer testable preview stable }.include?(status)
```

Comment:
Would we want to mandate a category for `internal` features, given that they are supposed to show up in internal UI?

Reply:
I'll follow @patrickangle's lead here. We can easily adjust later as needed.